operator/k8s_identity.go

```diff
@@ -64,12 +65,22 @@ func deleteIdentity(ctx context.Context, identity *v2.CiliumIdentity) error {
 
 var identityHeartbeat *identity.IdentityHeartbeatStore
 
+// counters for GC failed/successful runs
+var (
+	failedRuns     = 0
+	successfulRuns = 0
+)
+
 // identityGCIteration is a single iteration of a garbage collection. It will
 // delete identities that have not had its heartbeat lifesign updated since
 // option.Config.IdentityHeartbeatTimeout
 func identityGCIteration(ctx context.Context) {
 	log.Debug("Running CRD identity garbage collector")
 
+	identityStoreList := identityStore.List()
```

Comment:
This should be done after line 92

Reply:
I was putting these lines before the check `if identityStore == nil {...}`
I think now I get it, thank you :)